cpp/src/barretenberg/honk/composer/composer_helper/standard_honk_composer_helper.hpp

```diff
@@ -1,30 +1,28 @@
 #pragma once
 
-#include "barretenberg/polynomials/polynomial.hpp"
+#include <utility>
 #include "barretenberg/srs/reference_string/file_reference_string.hpp"
-#include "barretenberg/plonk/proof_system/proving_key/proving_key.hpp"
 #include "barretenberg/honk/proof_system/prover.hpp"
 #include "barretenberg/honk/proof_system/verifier.hpp"
 #include "barretenberg/proof_system/circuit_constructors/standard_circuit_constructor.hpp"
-#include "barretenberg/honk/pcs/commitment_key.hpp"
-#include "barretenberg/plonk/proof_system/verification_key/verification_key.hpp"
 #include "barretenberg/plonk/proof_system/verifier/verifier.hpp"
-#include "barretenberg/proof_system/arithmetization/gate_data.hpp"
 #include "barretenberg/proof_system/composer/composer_helper_lib.hpp"
 #include "barretenberg/proof_system/composer/permutation_helper.hpp"
 
-#include <utility>
+#include "barretenberg/honk/flavor/standard.hpp"
 
 namespace proof_system::honk {
-// TODO(Kesha): change initializations to specify this parameter
-// Cody: What does this mean?
-template <typename CircuitConstructor> class StandardHonkComposerHelper {
+class StandardHonkComposerHelper {
   public:
+    using Flavor = flavor::Standard;
```

Comment:
I'm a bit confused - I thought the idea is  we have a single  HonkComposer/HonkComposerHelper/CircuitConstructor interface with the `Flavour` telling us whether we use `Standard`, `Ultra` or `Turbo` implementation. Why do we need to keep this classes?

Reply:
We don't want a single class template that gives all circuit constructors, composers, provers, verifiers by specifying a single flavor template argument because then we end up with complicated conditional logic, like we have in the plonk prover, where (after non-trivial effort) you can find that one of the rounds is  actually a no-op except in the case of UltraPlonK. We're trying to strike a balance between explicitness and code reuse. I think we're a bit too far on the side of duplication at the moment, but the general direction is very good.